### PR TITLE
doc: fix ScalaFiddle integration for Akka 2.6

### PR DESCRIPTION
**Pull request by andreaTP**
## Purpose
Align Akka.Js to 2.6.1 to have working examples in the docs.
## References

References #27510

## Changes
Upgrade to latest published Akka.js version

### CODE REVIEW

**Comment by andreaTP** [comment]
To be finalized needs this to be solved:
https://github.com/scalafiddle/scalafiddle-core/issues/38

File: akka-docs/src/main/paradox/assets/js/scalafiddle.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 window.scalaFiddleTemplates = {
   "Akka": {
-    pre: "// $FiddleDependency org.akka-js %%% akkajsactor % 1.2.5.14 \n" +
-    "// $FiddleDependency org.akka-js %%% akkajsactorstream % 1.2.5.14 \n" +
-    "// $FiddleDependency org.akka-js %%% akkajsactortyped % 1.2.5.14 \n",
+    pre: "// $FiddleDependency org.akka-js %%% akkajsactor % 2.2.6.1 \n" +
+    "// $FiddleDependency org.akka-js %%% akkajsactorstream % 2.2.6.1 \n" +
+    "// $FiddleDependency org.akka-js %%% akkajsactortyped % 2.2.6.1 \n",
```
Comment (hepin1989) [inline akka-docs/src/main/paradox/assets/js/scalafiddle.js]: Cool

**Comment by andreaTP** [comment]
ready to be tested and to go in!

**Review by raboof — COMMENTED** [review]
Great! Tested and indeed works here as well.

We can remove the "FIXME: fiddle won’t work until Akka 2.6 is released and fiddle updated with that #27510" warning now then ;)

**Comment by raboof** [comment]
OK TO TEST

**Comment by andreaTP** [comment]
removed the FIXME

**Review by raboof — APPROVED** [review]

**Review by johanandren — APPROVED** [review]

**Pull request merged**